indra/newview/llappviewermacosx-objc.h

```diff
@@ -33,4 +33,6 @@
 //Why?  Because BOOL
 void launchApplication(const std::string* app_name, const std::vector<std::string>* args);
 
+void force_ns_sxeption() { [NSEXception raise:@"Forced NSException"] };
```

Comment:
mac build looks unhappy about this:
`/Users/runner/work/viewer/viewer/indra/newview/llappviewermacosx-objc.h:36:41: error: expected ',' or ']' in lambda capture list`
also
`/Users/runner/work/viewer/viewer/indra/newview/llappviewermacosx-objc.h:36:29: error: use of undeclared identifier 'NSEXception'`